va/va_cp_vp9.h

```diff
@@ -0,0 +1,121 @@
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+/**
+* \file va_cp_vp9.h
+* \brief The content protection API
+*
+* This file contains the \ref api_cp_vp9 "VP9 Content Protection API".
+*/
+
+#ifndef VA_CP_VP9_H
+#define VA_CP_VP9_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \defgroup api_cp_vp9 VP9 Content Protection API
+ *
+ * @{
+ */
+
+/**
+ * \brief uncompressed header for VP9 cenc decode
+ *
+ * This structure holds information for \c
+ * uncompressed_header() as defined by the VP9 specification.
+ *
+ */
+typedef struct _VACencHeaderParameterBufferVP9 {
+    
+    union {
+        struct {
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t profile                              : 2;            
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_existing_frame_flag             : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_to_show                         : 3;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t frame_type                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_frame                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t error_resilient_mode                 : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t color_space                          : 2; 
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_x                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_y                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t refresh_frame_flags                  : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t frame_parallel_decoding_mode         : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t render_and_frame_size_different      : 1;
+            /** \brief Reserved for future use, must be zero */
+            uint32_t reserved                             : 20;
+            } bits;
+        uint32_t value;
+    } header_fields; 
+     /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t frame_width_minus1;
+     /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t frame_height_minus1;
+    /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t render_width_minus1;
+    /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t render_height_minus1;
```

Comment:
This needs at least the `ref_frame_idx` array and the `intra_only` flag as well I think.

Reply:
Accepted.